build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```
@@ -279,6 +284,10 @@ fun Project.enableExperimentalTestFiltering() = !setOf("build-scan-performance",
 val isExperimentalTestFilteringEnabled
     get() = System.getProperty("gradle.internal.testselection.enabled").toBoolean()
 
+// Controls the test distribution partition size. The test classes smaller than this value will be merged into a "partition"
+val Project.maxTestDistributionPartitionSecond: Long?
+    get() = System.getProperty("maxTestDistributionPartitionSecond")?.toLong()
```

```suggestion
    get() = System.getProperty("testDistributionPartitionSizeInSeconds")?.toLong()
```

I'd call the variable differently.

---

Better now?